eq-author/src/App/page/Design/Validation/MetadataContentPicker.js

```diff
@@ -5,7 +5,7 @@ import { get } from "lodash";
 import ContentPickerSelect from "components/ContentPickerSelect";
 import { METADATA } from "components/ContentPickerSelect/content-types";
 
-import AvailableMetadataQuery from "App/questionPage/Design/Validation/AvailableMetadataQuery";
+import AvailableMetadataQuery from "App/page/Design/Validation/AvailableMetadataQuery";
```

Comment:
I think this could be 
```suggestion
import AvailableMetadataQuery from "./AvailableMetadataQuery";
```